src/transformers/models/owlv2/modeling_owlv2.py

```diff
@@ -1634,6 +1623,110 @@ def image_guided_detection(
             vision_model_output=vision_outputs,
         )
 
+    @add_start_docstrings_to_model_forward(OWLV2_IMAGE_GUIDED_OBJECT_DETECTION_INPUTS_DOCSTRING)
+    @replace_return_docstrings(output_type=Owlv2ImageGuidedObjectDetectionOutput, config_class=Owlv2Config)
+    def image_guided_detection_v2(
+        self,
+        pixel_values: torch.FloatTensor,
+        query_pixel_values: Optional[torch.FloatTensor] = None,
+        output_attentions: Optional[bool] = None,
+        output_hidden_states: Optional[bool] = None,
+        return_dict: Optional[bool] = None,
+    ) -> Owlv2ImageGuidedObjectDetectionOutput:
+        r"""
```

Comment:
The docstring should detail what the method does and how it differs from image_guided_detection